### ISSUE TITLE
Where is `_fds_forward` ?

### ISSUE THREAD

**Issue by mucunwuxian** [WildChlamydia/MiVOLO#31]
Thank you for your great article and repository.
I was impressed.

Please tell me about the code below.
https://github.com/WildChlamydia/MiVOLO/blob/main/mivolo/model/mivolo_model.py#L283

LDS is mentioned in the paper, but FDS seems not to be mentioned.
However, the code in the link above shows that FDS is applied.
Did you apply it on a trial basis while studying?
I looked for `_fds_forward` but couldn't find it.

**Comment by WildChlamydia** [comment]
Hello! Thank you very much for your words.

Yes, we indeed conducted experiments with the FDS technique as well, but none of our final models incorporate it. We did not observe significant improvement with its use, and the computational cost during the training stage is quite substantial, especially with large datasets. Consequently, we decided against it. I think we should remove it from the code. Thank you.
